frontend/src/components/crd/List.tsx

```diff
@@ -58,6 +58,13 @@ export default function CustomResourceDefinitionList() {
           getter: crd => crd.spec.scope,
           sort: true,
         },
+        {
+          label: t('Categories'),
+          getter: crd => {
+            const categories = crd.jsonData!.status.acceptedNames.categories;
+            return typeof categories !== 'undefined' ? categories : 'un';
```

Comment:
Why is the type being checked here? I think what's needed is `categories?.join(', ') || ''`. Because I don't think we have used this 'un' as a notation for undefined in other Headlamp areas.